tests/fips/mozilla_nss/firefox_nss.pm

```diff
@@ -68,6 +71,19 @@ sub run {
     my $firefox_version = script_output(q(rpm -q MozillaFirefox | awk -F '-' '{ split($2, a, "."); print a[1]; }'));
     record_info('MozillaFirefox version', "Version of Current MozillaFirefox package: $firefox_version");
 
+    if (get_var('FIPS_ENABLED')) {
+        zypper_call('in mozilla-nss mozilla-nss-certs');
+        my $mozilla_nss_ver = script_output("rpm -q --qf '%{version}\n' mozilla-nss");
```

Comment:
@jouyingbin  I suggest using 'package_version_cmp', but not must.

Reply:
ah yes, it is a nicer way :) I will modify it, thanks.

Reply:
change to package_upgrade_check()